src/panels/lovelace/editor/picture-element-editor/hui-picture-element-element-editor.ts

```diff
@@ -0,0 +1,52 @@
+import { customElement } from "lit/decorators";
+import { LovelaceElementConfig } from "../../elements/types";
+import type { LovelacePictureElementEditor } from "../../types";
+import { HuiElementEditor } from "../hui-element-editor";
+import "../config-elements/elements/hui-state-badge-element-editor";
+import "../config-elements/elements/hui-state-icon-element-editor";
+import "../config-elements/elements/hui-state-label-element-editor";
+import "../config-elements/elements/hui-service-button-element-editor";
+import "../config-elements/elements/hui-icon-element-editor";
+import "../config-elements/elements/hui-image-element-editor";
+import "../config-elements/elements/hui-conditional-element-editor";
+
+@customElement("hui-picture-element-element-editor")
+export class HuiPictureElementElementEditor extends HuiElementEditor<LovelaceElementConfig> {
+  protected get configElementType(): string | undefined {
+    return this.value?.type;
+  }
+
+  protected async getConfigElement(): Promise<
```

Comment:
This function should be part of the elements, so custom elements can also provide an editor

Reply:
Think I've got this resolved.